src/rpm/headers/header.rs

```diff
@@ -191,67 +191,67 @@ where
             })
     }
 
-    pub(crate) fn get_entry_i16_array_data(&self, tag: T) -> Result<Vec<i16>, RPMError> {
+    pub(crate) fn get_entry_data_as_u16_array(&self, tag: T) -> Result<Vec<u16>, RPMError> {
         let entry = self.find_entry_or_err(&tag)?;
         entry
             .data
-            .as_i16_array()
+            .as_u16_array()
             .ok_or_else(|| RPMError::UnexpectedTagDataType {
-                expected_data_type: "i64 array",
+                expected_data_type: "uint64 array",
```

Comment:
I'm conflicted about tweaking the names from the documented ones even if it makes it clearer.  Thoughts?